functions/src/routes/experiences/getExperiences.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable promise/no-nesting */
```

Comment:
This one eh

Reply:
It can all be filtered on the frontend actually.
- `happeningNow`: After fetching all experiences, you can filter then by checking is the start date of the event is `<` the current date-time.
- `nearby places`: We can make the address selection be from Google Maps, like how we type addresses on uber and it gives us suggestions from google maps, so that's what we'll store in the database, so when fetching for nearby places, we use the location stored and their current location along with the google maps API to check for proximity.
- `upcomingEvents`: It's just like happeningNow, but rather with a `>` check between the current time and the event startTime.

Reply:
I was thinking it would be better to have all these on the backend, it makes the flow easier.